### PR TITLE
Updates to OptimizeStartRegimens, vlOutcomeAndUptake, AggregateTimeToARTGrp and DimPatient

### PR DESCRIPTION
**Pull request by cbrianbet**
Added startart brakedowns to AggregateUptakeOutcome
Standardized gender in DimPatient
Updated logic for Last12MVLResult

### CODE REVIEW

File: Scripts/REPORTING/load_aggregate_OptimizeStartRegimens.sql [hunk 1]
```diff
@@ -1,45 +1,75 @@
...
+		Gender,
+		CASE
+			
+			WHEN floor( age.Age ) < 15 THEN
+			'Child' 
+			WHEN floor( age.Age ) >= 15 THEN
+			'Adult' ELSE 'Aii' 
+		END AS Agegroup,
+		DATIMAgeGroup,
+		CASE
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_OptimizeStartRegimens.sql:50]: @cbrianbet perhaps we can also move this definition upstream to the fact ?

**Review by nobert-mumo — DISMISSED** [review]

**Review by Marymary-dev — APPROVED** [review]

**Review by Marymary-dev — APPROVED** [review]

File: Scripts/REPORTING/load_aggregate_OptimizeStartRegimens.sql (hunk 1 above)
Comment (cbrianbet) [reply Scripts/REPORTING/load_aggregate_OptimizeStartRegimens.sql:44]: @nobert-mumo changes have now been addressed

File: Scripts/REPORTING/load_aggregate_OptimizeCurrentRegimens.sql [hunk 2]
```diff
@@ -2,72 +2,114 @@
...
-	-- LEFT JOIN NDWH.dbo.DimPatient p on ART.PatientPK = p.PatientPK and ART.SiteCode = P.SiteCode
+		CASE 
+			WHEN cast (age as float) <2 THEN '<2 Years'
+			WHEN cast (age as float) between 2 and 4	THEN '2-4 Years'
+			WHEN cast (age as float) between 5 and 9 THEN '5-9 Years'
+			WHEN cast (age as float) between 10 and 14 THEN '10-14 Years'
+			ELSE 'Adults' 
+		END AS AgeBands,
+    
+		CASE
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_OptimizeCurrentRegimens.sql:70]: @cbrianbet just noticed this; would it be better to store this in a look up table and join on it ? :) 
I think it would be easier to manage.

Comment (cbrianbet) [reply Scripts/REPORTING/load_aggregate_OptimizeCurrentRegimens.sql:70]: Like its own Dim?

Comment (cbrianbet) [reply Scripts/REPORTING/load_aggregate_OptimizeCurrentRegimens.sql:70]: I think its only being used in the Optimize Regimen tables 

> @cbrianbet just noticed this; would it be better to store this in a look up table and join on it ? :) I think it would be easier to manage.

**Comment by nobert-mumo** [comment]
cbrianbet fair enough, though its a bit hard to follow especially for the ones that are a long list. But also this seems like a leakage from the data cleaning scripts, we can revisit this later
